admin-api/resource_server/api-platform.md

```diff
@@ -0,0 +1,65 @@
+---
+title: API Platform and CQRS integration
+menuTitle: API Platform and CQRS
+weight: 1
+---
+
+# API Platform and CQRS integration
+
+## CQRS based API
+
+Since the beginning of the Symfony migration we whose CQRS as our base architecture, and their usage for the future API was already anticipated because:
```

Comment:
```suggestion
Since the beginning of the Symfony migration, we chose CQRS as our base architecture, and their usage for the future API was already anticipated because:
```